test/test_models.py

```diff
@@ -782,7 +783,8 @@ def test_detection_model(model_fn, dev):
     model.eval().to(device=dev)
     x = _get_image(input_shape=input_shape, real_image=real_image, device=dev)
     model_input = [x]
-    out = model(model_input)
+    with torch.no_grad(), freeze_rng_state():
```

Comment:
Thanks, this definitely make sense to use `torch.no_grad()` and `freeze_rng_state()` since we dont do any backprop test. Should we also add this when we test with autocast below (line 849)?

Reply:
You are right, just added them :+1: